README.md

```diff
@@ -35,3 +35,6 @@ docker run -e DATABRICKS_HOST=$YOUR_HOST_URL -e DATABRICKS_TOKEN=$YOUR_TOKEN ghc
 This CLI follows the Databricks Unified Authentication principles.
 
 You can find a detailed description at https://github.com/databricks/databricks-sdk-go#authentication.
+
+## Privacy Notice
+Databricks CLI is included in our definition of data processing platform products and services (the "Platform services") and subject to the [Databricks Privacy Notice](https://www.databricks.com/legal/privacynotice).
```

Comment:
```suggestion
Databricks CLI is included in the definition of data processing platform products and services (the "Platform services") and subject to the [Databricks Privacy Notice](https://www.databricks.com/legal/privacynotice).
```
Very minor, but in the docs our standard style is to never use plural first person.

Reply:
Thanks Julia; the updated recommendation from legal no longer uses the first person,though.